enterprise/cli/dbcrypt_rotate.go

```diff
@@ -0,0 +1,152 @@
+//go:build !slim
+
+package cli
+
+import (
+	"bytes"
+	"context"
+	"database/sql"
+	"encoding/base64"
+
+	"cdr.dev/slog"
+	"cdr.dev/slog/sloggers/sloghuman"
+	"github.com/coder/coder/v2/cli"
+	"github.com/coder/coder/v2/cli/clibase"
+	"github.com/coder/coder/v2/coderd/database"
+	"github.com/coder/coder/v2/codersdk"
+	"github.com/coder/coder/v2/enterprise/dbcrypt"
+
+	"golang.org/x/xerrors"
+)
+
+func (*RootCmd) dbcryptRotate() *clibase.Cmd {
+	var (
+		vals = new(codersdk.DeploymentValues)
+		opts = vals.Options()
+	)
+	cmd := &clibase.Cmd{
+		Use:   "dbcrypt-rotate --postgres-url <postgres_url> --external-token-encryption-keys <new-key>,<old-keys>",
```

Comment:
nit: it could be a new fancy command: `coder encryption rotate-database-keys` :)

Reply:
I can make `rotate` a subcommand 👍  This will make it easier to add a `decrypt` command in future.

Reply:
I don't think dbcrypt stuff should be top-level, it should be `coder server dbcrypt rotate` or something. And you can add other subcommands related to dbcrypt under that. We keep all the server-related stuff like `postgres-builtin-url` etc. under the `server` subcommand currently